client/js/components/addon/AddonWrapper.vue

```diff
@@ -0,0 +1,77 @@
+<license>
+  (c) 2010-present DEMOS E-Partizipation GmbH.
+
+  This file is part of the package demosplan,
+  for more information see the license file.
+
+  All rights reserved
+</license>
+
+<template>
+  <div>
+    <component
+      :is="component"
+      v-bind="getCurrentProperties()"
+    />
+  </div>
+</template>
+
+<script>
+import { checkResponse, dpRpc } from '@demos-europe/demosplan-utils'
+export default {
+  name: 'AddonWrapper',
+
+  props: {
+    hookName: {
+      type: String,
+      required: true,
+      default: ''
+    },
+
+    addonProps: {
+      type: Object,
+      required: false,
+      default: () => {}
+    }
+  },
+
+  data () {
+    return {
+      component: ''
+    }
+  },
+
+  methods: {
+    getCurrentProperties() {
+      return this.addonProps
+    },
+
+    loadComponents(hookName) {
+      const params = {
+        hookName: hookName
+      }
+
+      dpRpc('addons.assets.load', params)
+        .then(response => checkResponse(response))
+        .then(response => {
+          const result = response[0].result
+
+          for (const key of Object.keys(result)) {
+            const addon = result[key]
+            const contentKey = addon.entry + '.umd.js'
+            const content = addon.content[contentKey]
+
+            eval(content)
```

Comment:
Just to ask this here once: we are 100% sure that `content` can be considered "safe" at this point?